File: backend/Space.Card.Game.WebApi/Dtos/BattleCommandRequestDto.cs
using Space.Card.Game.WebApi.Interfaces.Commands;
using Space.Card.Game.WebApi.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Space.Card.Game.WebApi.Dtos
{
    public class BattleCommandRequestDto : IBattleCommandRequest
    {
        public int StarshipOneId { get; set; }
        public int StarshipTwoId { get; set; }
    }
}

File: frontend/src/app/dtos/requests/battle-command-request.dto.ts
export class BattleCommandRequestDto {

  constructor(starshipOneId: number, starshipTwoId: number) {
    this.starshipOneId = starshipOneId;
    this.starshipTwoId = starshipTwoId;
  }

  starshipOneId: number;
  starshipTwoId: number;
}
